### ISSUE TITLE
Schema does not properly validate $action_wml and $conditional_wml

### ISSUE THREAD

**Issue by soliton-** [wesnoth/wesnoth#7857]
- `$action_wml` and `$conditional_wml` contain `any_tag=yes` which was presumably supposed to be removed in 7ab5fab
- they contain tag names with commas that are simply invalid currently

For the latter it'd be nice to support commas in tag names so the current schema just works.

`any_tag` should get removed from `$action_wml` and `$conditional_wml` and remaining schema issues fixed.

Somewhat unrelated: it'd be nice to consolidate "virtual" tags like `$action_wml` or `~generic~` to just use `~`. Simpler if there is only a single convention and `$` looks more like a variable which seems more confusing than `~` IMO.

**Comment by CelticMinstrel** [comment]
> * which was presumably supposed to be removed in [7ab5fab](https://github.com/wesnoth/wesnoth/commit/7ab5fabbf24976a18e6acb92a270dc0aa3d2ab35)

No, it's not supposed to be removed, ever. If you remove it, campaigns will fail to validate because they use custom tags.

**Comment by soliton-** [comment]
Why would campaigns with custom tags not provide the schema for those tags? Isn't that what #7794 is all about?

**Comment by gfgtdf** [comment]
#7794 is not implemented yet, so we clearly cannot remove the `any_tag=yes` yet.

Also its questionable whether we should force users to write a schema for their tags (on the other hand, they can probably just define their own custom tags as "everything allowed inside", so it's not that hard.)

**Comment by CelticMinstrel** [comment]
> Why would campaigns with custom tags not provide the schema for those tags? Isn't that what #7794 is all about?

Even if schema extensions are implemented, not providing one shouldn't cause schema validation to fail. Providing one just allows you to catch additional errors that it wouldn't spot otherwise.

> on the other hand, they can probably just define their own custom tags as "everything allowed inside", so it's not that hard.

Sure, but why bother if that's what you're going to do? That's the same as not providing one at all.

**Comment by soliton-** [comment]
The invalid schema entries were fixed in 3c45a5b0e72a2b61dded8db149994652f8035f88 and the schema validation in our CI was fixed with 2f8d471e8056832df73228021b3d7810ccb17f3f.

Hopefully something more useful for UMC schema validation will come after #7794 is implemented.
